ChangeLog.d/MD-X.509-hashing.txt

```diff
@@ -0,0 +1,2 @@
+Features
+   * x.509 now uses PSA for hashing (conditionally on MBEDTLS_USE_PSA_CRYPTO).
```

Comment:
Changelong entries are intended for users of the library, not for developers. So they should explain changes in terms of what's visible for users. The fact that part of the X.509 module is written in a different way is not relevant for users. However, this change has a consequence for users: X.509 code will now make use of PSA acceerator code if present.

Also, please use proper English grammar, spelling, punctuation, capitalization, etc. in changelog entries. (In git commit messages, it's still appreciated, but much less important.)

Thus, suggested wording:

> The X.509 module now uses PSA hash acceleration if present.